Formula/sifu.rb

```diff
@@ -0,0 +1,22 @@
+class Sifu < Formula
+  desc "Rapid software development tool made for creating cloud solutions."
+  homepage "https://codesifu.com"
+  url "https://codesifu.com/download/sifu-0.1.1.tar.gz"
+  sha256 "d0ef00a475766bf4717b4721044bdbd0f1c8c02be3a1f824738d306b08a06cd2"
+
+  depends_on :java
+
+  def install
+    rm_f Dir["*.bat"]
+    libexec.install Dir["*"]
+
+    (bin/"sifu").write <<-EOS.undent
+      #!/bin/sh
+      exec "#{libexec}/sifu" "$@"
+    EOS
+  end
+
+  test do
+    system "sifu", "--help"
```

Comment:
Hi, in case of this tool, i can't do anything smarter in test. 
After starting the tool there is a question that need to be answered and i'm not sure can i do that trough formula.

> Regardless, need "#{bin}/sifu"
I'm not sure that i understand this...